exercises/practice/etl/data.csv

```diff
@@ -0,0 +1,4 @@
+"{""1"":[""A""]}",""
```

Comment:
For the other maintainers, what are you thoughts on what is essentially a JSON exercise on a SQL track?

Reply:
I’m on the fence. It’s not a great fit since we’re wrangling JSON at the end of the day and not SQL data. The input data makes sense to me as a data structure but not a table of data. There are a limited number of ways to solve it too.

Reply:
My opinion is, since this is an SQLite track, we're trying to teach about its features, so doesn't it make sense to have an exercise where you practice handling JSON? Because JSON is actually used to store values in databases.